### PR TITLE
Add toggle case and printify in Edit menu

### PR DESCRIPTION
**Pull request by kelsa-pi**

### CODE REVIEW

**Comment by almarklein** [comment]
@kelsa-pi Thanks for the contribution!

I think the toggle case command is a valuable addition, but I am less sure about the printify action, to be honest. It targets a very specific use-case, and IMO it is not worth the extra menu item. Someday, we'll have a plugin system that should allow specific actions such as this :)

**Comment by kelsa-pi** [comment]
@almarklein 
Toggle case command as new menu item is OK  :)

**Comment by kelsa-pi** [comment]
I hope it's OK.

File: pyzo/core/menu.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
++# -*- coding: utf-8 -*-
```
Comment (almarklein) [inline pyzo/core/menu.py]: ?

File: pyzo/codeeditor/base.py [hunk 2]
```diff
@@ -867,4 +867,32 @@ def updateMargins(self):
         accordingly.
         """
         leftmargin = self.getLeftMargin()
-        self.setViewportMargins(leftmargin , 0, 0, 0)
+        self.setViewportMargins(leftmargin , 0, 0, 0
```
Comment (almarklein) [inline pyzo/codeeditor/base.py]: This looks like a syntax error.

**Comment by almarklein** [comment]
Thanks, I added two minor comments.

**Comment by almarklein** [comment]
If you address the two minor comments, I can merge this before the next release :)

**Comment by almarklein** [comment]
I just went ahead and merged this. I fixed the two minor issues in a new commit. Thanks for this contribution!

**Pull request merged**